analysis/topEFT/run_topcoffea.py

```diff
@@ -102,6 +110,13 @@
     # If we don't specify this argument, it will be None, and the processor will fill all hists
     hist_lst = args.hist_list
 
+if executor == "futures":
+    if dotest:
```

Comment:
I think this could have unintuitive behavior if the user asks for the wq executor and the `test` option (looks like the code will just silently not actually run in `test` mode).

However, I'm not sure this `dotest` functionality is actually used. I never use it anyway. When I want to run a test, I just explicitly set the number of chunks and/or number of cores myself. Does anyone else (e.g. @bryates) actually use this option? If it's never used, I'd be in favor of just removing it entirely. And I think the `pretend` option is similar. Unless someone actually uses that, I think we should just remove the option.

Reply:
Yes, I've used `-t` in the past, but only locally. I agree in principle that it could cause issues on WQ, but I don't know if we'd ever ask WQ to use `-t`, so it should be fine.

Reply:
Ok @bryates if you use the option I agree we should leave it in. I've added a check to give an error if we try to use that option with anything other than futures. I also added a check to throw an error if someone asks for an executor other than futures or work_queue (since the run script is just set up to run with those two executors).

Reply:
Looks good to me.